healpix_alchemy/point.py

```diff
@@ -61,16 +74,10 @@ def within(self, other, radius):
         bool
 
         """
-        sin_radius = sind(radius)
-        cos_radius = cosd(radius)
+        sin_radius = sin(rad(radius))
+        cos_radius = cos(rad(radius))
         carts = (obj.cartesian for obj in (self, other))
-        terms = ((lhs.between(rhs - 2 * sin_radius, rhs + 2 * sin_radius),
+        terms = ((between(lhs, rhs - 2 * sin_radius, rhs + 2 * sin_radius),
```

Comment:
Not part of this PR really, but maybe some pointer to the expression sources?

Reply:
Isn't it clear what it's doing from the name of the `bounding_box_terms` variable?

Reply:
Sure. Just might deserve a comment, especially as that line is just "terms"

Reply:
Done, see 0c395b9e14c1bceb714e2e27a8a28325b0bd18a4.

Reply:
Looks good to me.